java-checks/src/test/java/org/sonar/java/checks/CatchOfThrowableOrErrorCheckTest.java

```diff
@@ -22,12 +22,14 @@
 import org.junit.jupiter.api.Test;
 import org.sonar.java.checks.verifier.JavaCheckVerifier;
 
+import static org.sonar.java.checks.verifier.TestUtils.nonCompilingTestSourcesPath;
+
 class CatchOfThrowableOrErrorCheckTest {
```

Comment:
I feel we are missing the test of the absence of semantic, aren't we?
If I add the following test, it fails with what looks to me like FPs.
```
  @Test
  void testNoSemantic() {
    JavaCheckVerifier.newVerifier()
      .onFile(nonCompilingTestSourcesPath("checks/CatchOfThrowableOrErrorCheck.java"))
      .withCheck(new CatchOfThrowableOrErrorCheck())
      .withClassPath(Collections.emptyList())
      .verifyIssues();
  }
```

Reply:
Good catch, I will look into that